### ISSUE TITLE
Nested @import rules and relative module resolution

### ISSUE THREAD

**Issue by timkelty** [TrySound/postcss-easy-import#24]
See https://github.com/tailwindcss/tailwindcss/issues/428

I'm trying to do this:

```
@responsive {
  @import 'suitcss-components-grid';
}
```
- `@responsive` being an at-rule provided by https://github.com/tailwindcss/tailwindcss
- `suitcss-components-grid` being an import from node_modules

However, when it tries to resolve, it always looks locally:
```
ERROR in ./node_modules/css-loader??ref--2-1!./node_modules/postcss-loader/lib??ref--2-2!./src/css/index.scss
Module not found: Error: Can't resolve './suitcss-components-grid' in '/Users/timkelty/Sites/dlr-craft3/src/css'
resolve './suitcss-components-grid' in '/Users/timkelty/Sites/dlr-craft3/src/css'
  using description file: /Users/timkelty/Sites/dlr-craft3/package.json (relative path: ./src/css)
```

I'm thinking this is because of [this behavior from postcss-import](https://github.com/postcss/postcss-import#root):
> Note: nested @import will additionally benefit of the relative dirname of imported files.

Any ideas?

**Comment by simonsmith** [comment]
I'm guessing this happens when the `@import` is nested in any ruleset? Or just at rules?

**Comment by timkelty** [comment]
I can take the path resolution out of the equation by just using a local file.
When I do that: `@responsive` is correctly making media rules, but their selectors are not prefixed.

`@responsive` acts on the contents of its rule, so basically I need a way to ensure the contents are imported/replaced before `@responsive` handles them.

Not sure if the solution would lie in `postcss-easy-import`, or in Tailwind's `@responsive` rule.

**Comment by timkelty** [comment]
> I'm guessing this happens when the @import is nested in any ruleset? Or just at rules?

As far as the relative module resolution, yes.

**Comment by simonsmith** [comment]
This plugin is a wrapper around [postcss-import](https://github.com/postcss/postcss-import) so it might be worth trying that plugin and seeing if you can reproduce the error.

**Comment by timkelty** [comment]
@simonsmith good idea - I'll try that. I started here because it seemed specific to non-relative (node_modules) imports, but I probably need some more repro-cases.

**Comment by AlexWayfer** [comment]
Doesn't convert nested imports for me at all:

`main.css`:

```pcss
@import 'foo';

body {
  @import 'bar';
}
```

`foo.css`:

```pcss
p {
  color: red;
}
```

`bar.css`:

```pcss
button {
  color: blue;
}
```

Output:

```css
p {
  color: red;
}

body {
  @import 'bar';
}
```

**Comment by LeoColomb** [comment]
Related:
* https://github.com/postcss/postcss-import/issues/390
* https://github.com/postcss/postcss-import/issues/296
